CMakeLists.txt

```diff
@@ -2,13 +2,21 @@ project(ffxiv-wireshark-dissector C)
 
 cmake_minimum_required(VERSION 3.7)
 set(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake)
-set(CMAKE_INSTALL_LIBDIR ~/.wireshark)
+IF (WIN32)
+  set(CMAKE_INSTALL_LIBDIR $ENV{APPDATA}/Wireshark/plugins)
```

Comment:
The changes to CMakeLists.txt are only necessary if compiling as an external library. Basically as an external library you need to tell CMake where all the include directories are as well as the library directories are. If we're not supporting compilation as an external library for Windows then yes it's not needed.